docs/getting_started.md

```diff
@@ -0,0 +1,166 @@
+In most scenarios, we recommend starting with the `filtered.rds` file. 
+
+The first step in analyzing the provided gene expression data will be to import the data into R. 
+As a reminder, each RDS file contains a [`SingleCellExperiment` object](https://bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html). 
+Refer to {ref}`single-cell gene expression file contents<sce_file_contents:single-cell gene expression file contents>` for a more detailed description on the contents of the included `SingleCellExperiment` object. 
+
+In order to work with `SingleCellExperiment` objects in R, we need to ensure that we have the [`SingleCellExperiment` package]() installed and loaded. 
+
+The below commands can be used to import the RDS file into R and save the `SingleCellExperiment` object. 
+
+```r
+# if SingleCellExperiment is not installed, install the package
+# otherwise the installation step can be skipped
+BiocManager::install("SingleCellExperiment")
+
+library(SingleCellExperiment)
+# read in the RDS file, including the complete path to the file's location
+sce <- readRDS("SCPCL000000_filtered.rds")
+```
+
+More resources for learning about `SingleCellExperiment` objects: 
+
+- [`SingleCellExperiment` objects Vignette](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) on working with `SingleCellExperiment` objects
+- [Orchestrating Single Cell Analysis chapter on the `SingleCellExperiment` class](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html)
+
+## Quality control  
+
+After we have imported the RDS file into R and accessed the `SingleCellExperiment` object, we can begin working with the data. 
+Before we perform any downstream steps, it is recommended to remove any low quality cells from our dataset.
+This would include cells that may be dying or damaged, showing a higher percentage of reads coming from mitochondrial cells. 
+Low quality cells may also be present due to technical artifacts such as inefficient reverse transcription or PCR amplification resulting in a lower number of total reads and unique genes identified. 
+
+For the filtered `SingleCellExperiment` object present in all `filtered.rds` files, we have used [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a data driven approach to predict low-quality cells, to identify cells that should be removed prior to downstream analyses. 
+`miQC` jointly models the proportion of mitochondrial reads and the number of unique genes detected in each cell to calculate the probability of a cell being compromised (i.e. dead or damaged). 
+High-quality cells are those with a low probability of being being compromised (< 0.75) or sufficiently low mitochondrial content. 
+We have already incorporated the `miQC` results into the {ref}`colData() of the SingleCellExperiment object<sce_file_contents:cell metrics>`. 
```

Comment:
I would maybe move this link to the `colData()` section to the sentence in my first suggestion.